src/integrationTest/java/uk/gov/hmcts/reform/wacaseeventhandler/controllers/ReceivedMessagesHealthControllerTest.java

```diff
@@ -129,6 +133,85 @@ void test_health_reports_up_if_time_outside_of_working_hours_end_time() throws E
         verify(caseEventMessageRepository, never()).getNumberOfMessagesReceivedInLastHour(any());
     }
 
+    @Test
+    @Sql(executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD,
+        scripts = {"classpath:scripts/insert_case_event_messages_for_received_messages_check.sql"})
+    void test_health_reports_up_if_received_messages_in_last_hour() throws Exception {
```

Comment:
can we update this to use Should_ExpectedBehavior_When_StateUnderTest naming convention?